README.md

```diff
@@ -8,7 +8,7 @@ ABOUT
 This is a small application that sits in the system tray and updates a text
 file with the currently playing audio track.
 
-It supports Spotify, iTunes, Winamp, foobar2000, and VLC.
+It supports Spotify, iTunes, Winamp, foobar2000, VLC and mps-youtube.
```

Comment:
I prefer Chicago style of writing and like a comma in my list.  VLC, and mps-youtube.